code/datums/emergency_calls/clf.dm

```diff
@@ -1,73 +1,57 @@
 	probability = 20
 
 
-/datum/emergency_call/clf/print_backstory(mob/living/carbon/human/mob)
-	to_chat(mob, "<B>You grew up [pick(5;"on the TGMC prison station", 10;"in the LV-624 jungle", 25;"on the farms of LV-771", 25;"in the slums of LV-221", 20;"the red wastes of LV-361", 15;"the icy tundra of LV-571")] to a [pick(50;"poor", 15;"well-off", 35;"average")] family.</B>")
-	to_chat(mob, "<B>As a native of the Tychon's Rift sector, you joined the CLF because [pick(20;"the Dust Raiders killed someone close to you in 2181", 20;"you harbor a strong hatred of the TerraGov", 10;"you are a wanted criminal in the TerraGov", 5;"have UPP sympathies and want to see the TGMC driven out of the secor", 10;"you believe the TGMC occupation will hurt your quality of life", 5;"are a violent person and want to kill someone for the sake of killing", 20;"want the Tychon's Rift to be free from outsiders", 10;"your militia was absorbed into the CLF")] and are considered a terrorist by the TGMC.</B>")
-
-	to_chat(mob, "<B>The Tychon's Rift sector has largely enjoyed its indepdendence..</B>")
-	to_chat(mob, "<B>Though technically part of the TerraGov frontier, many colonists in the Tychon's Rift have enjoyed their freedoms.</B>")
-	to_chat(mob, "")
-	to_chat(mob, "<B>In 2181, however, the TerraGov moved the TGMC Battalion, the 'Dust Raiders', and the battalion flagship, the USS Alistoun, to the Tychon's Rift sector. </B>")
-	to_chat(mob, "<B>The Dust Raiders responded with deadly force, scattering many of the colonists who attempted to fight their occupation.</B>")
-	to_chat(mob, "<B>The Dust Raiders and their flagship, the USS Alistoun eventually withdrew from the sector by the end of the year.</font></B>")
-	to_chat(mob, "<B> ")
-	to_chat(mob, "<B>With the Tychon's Rift sector existing in relative isolation from TerraGov oversight for the last five years, many colonists have considered themselves free from governmental rule.</B>")
-	to_chat(mob, "")
-	to_chat(mob, "<B>The year is now 2186.</B>")
-	to_chat(mob, "<B>The arrival of the TGMC Battalion, the Falling Falcons, and their flagship, the USS Almayer, have reaffirmed that the TerraGov considers Tychon's Rift part of their holdings.</B>")
-	to_chat(mob, "<B>It is up to you and your fellow colonists to make them realize their trespasses. This sector is no longer theirs.</B>")
+/datum/emergency_call/clf/print_backstory(mob/living/carbon/human/H)
+	to_chat(H, "<B>You grew up [pick("on the TGMC prison station","in the LV-624 jungle","on a desert planet","on an icy colony")] to a[pick(50;" poor", 15;" well-off", 35;"n average")] family.</B>")
+	to_chat(H, "<B>You joined the CLF because [pick(20;"you harbor a strong hatred of the TerraGov",5;"you are a violent person and want to kill someone for the sake of killing", 10;"your militia was absorbed into the CLF")] and are considered a terrorist by the TGMC.</B>")
```

Comment:
```suggestion
	to_chat(H, "<B>You joined the CLF because [pick(20;"you harbor a strong hatred for the oppressive TerraGov",5;"you are good at killing, and in times like these this is the place to be", 10;"your militia was absorbed into the CLF")] and you are considered a terrorist by the TGMC.</B>")
```